dev/io.openliberty.security.jakartasec.3.0.internal.cdi/src/io/openliberty/security/jakartasec/cdi/beans/OidcHttpAuthenticationMechanism.java

```diff
@@ -136,7 +133,7 @@ public AuthenticationStatus validateRequest(HttpServletRequest request,
         } else if (isCallbackRequest(request)) {
             status = processCallback(client, request, response, httpMessageContext);
         } else if (alreadyAuthenticated) {
-            status = processExpiredTokenResult(processExpiredToken(client, request, response), httpMessageContext);
+            status = processExpiredTokenResult(processExpiredToken(client, request, response), client, request, response, httpMessageContext);
```

Comment:
**nit:** don't have to pass request and response as params, since they can be gotten from the `HttpMessageContext`